addons/medical_vitals/functions/fnc_updateHeartRate.sqf

```diff
@@ -46,6 +46,7 @@ private _heartRate = GET_HEART_RATE(_unit);
 
 if !IN_CRDC_ARRST(_unit) then {
```

Comment:
Isn't this preventing the unit from entering when in cardiac arrest?

> On the first tick of handle vitals once CPR was successfully completed the function would be entered with a HR of 40 and the target HR would be 80. This would make the function
>40 + ((80 - 40) / 2 )*10 = 240, which would be a HR high enough to push the unit back into cardiac arrest.

Reply:
That is only resetting the statemachine though and not resetting the variables such as openWounds right? My worry is that if the new unit does not override the variables from the old unit on respawn it will have a reference to the actual arrays meaning they will receive any updates to the old arrays (and therefor receive wounds when shooting the old corpse).